docs/source/configuration/key_value_service_configs/postgres_key_value_service_config.rst

```diff
@@ -60,8 +60,64 @@ A minimal AtlasDB configuration for running against postgres will look like :
 
 For more details on the leader block, see :ref:`Leader Configuration <leader-config>`.
 
-Connection parameters
----------------------
+Hikari Connection parameters
+----------------------------
+
+We also provide the following options, which are mapped to `Hikari connection options <https://github.com/brettwooldridge/HikariCP#configuration-knobs-baby>`__:
+
+.. list-table::
+    :widths: 20 20 20 80
+    :header-rows: 1
+
+    *    - Option
+         - Default
+         - Hikari config equivalent
+         - Description
+
+    *    - ``minConnections``
+         - 8
+         - ``minimumIdle``
+         - The number of connections in an idle Hikari pool.
+
+    *    - ``maxConnections``
+         - 256
+         - ``maximumPoolSize``
+         - The maximum size the pool is allowed to reach.
+
+    *    - ``maxConnectionAge``
+         - 1000
```

Comment:
I think this is `1800`? https://github.com/palantir/atlasdb/blob/develop/atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java#L71

Reply:
Oh, very well spotted! 

*runs to shop, buys cap, comes back, doffs cap*